vignettes/demo.Rmd

```diff
@@ -49,6 +49,8 @@ pkg <- usethis::create_package("tea")
 ```{r create-package-real, echo = FALSE}
 pkg_dir <- file.path(tempdir(), "fledge")
 dir.create(pkg_dir, recursive = TRUE)
+dir.create(file.path(pkg_dir, "remote"))
+gert::git_init(file.path(pkg_dir, "remote"), bare = TRUE)
```

Comment:
Does the remote need to sit underneath `pkg_dir`? Perhaps it's safer to move it outside?

Reply:
It's a misleading name, the remote is _next_ to the actual `tea/` folder.

Reply:
I accept name suggestions :grin: